packages/backend-function/src/factory.ts

```diff
@@ -209,14 +222,43 @@ class AmplifyFunction
   readonly resources: FunctionResources;
   constructor(scope: Construct, id: string, props: HydratedFunctionProps) {
     super(scope, id);
+    const functionLambda = new NodejsFunction(scope, `${id}-lambda`, {
+      entry: props.entry,
+      environment: props.environment as { [key: string]: string }, // for some reason TS can't figure out that this is the same as Record<string, string>
+      timeout: Duration.seconds(props.timeoutSeconds),
+      memorySize: props.memoryMB,
+      runtime: nodeVersionMap[props.runtime],
+      bundling: {
+        // Added '\' to the end of each line for readability here
+        // eslint-disable-next-line spellcheck/spell-checker
```

Comment:
eslint was complaining about `ssm` here, let me know if this should be added to the dictionary.